frontend/src/constants/routes.js

```diff
@@ -31,3 +32,9 @@ export const PARTY_PROFILE = {
   dynamicRoute: (id) => `/profile/${id}`,
   component: PartyProfile,
 };
+
+export const RELATIONSHIP_PROFILE = {
+  route: "/dashboard/:id/contact-information/:typeCode",
```

Comment:
is`RELATIONSHIP_PROFILE` replacing the `PARTY_PROFILE`? - 
or is this the history profile?

Reply:
RELATIONSHIP_PROFILE is the history of the contact type (Mine Manager, Permittee, etc). eg: http://localhost:3000/dashboard/a083e796-651d-47d5-9788-74d76fbbae37/contact-information/MOR is the history of all mine managers for that mine.

Reply:
got it, thanks!